lib/src/components/NxTransferList/TransferListHalf.scss

```diff
@@ -118,6 +127,43 @@ $-visible-item-count: 10;
   }
 }
 
+.nx-transfer-list__item--with-reordering {
+  .nx-transfer-list__select {
+    border-radius: var(--nx-border-radius);
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+  }
+}
+
+.nx-transfer-list__button-bar {
+  border: var(--nx-border-default);
+  border-left: 0;
+  border-radius: var(--nx-border-radius);
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+  gap: var(--nx-spacing-base);
+  margin: 0;
+  padding: var(--nx-spacing-base) var(--nx-spacing-2x);
+}
+
+$-button-size: 32px;
+
+.nx-transfer-list__button {
```

Comment:
IMO this class doesn't need to exist, you can just select `.nx-btn` within `.nx-transfer-list__button-bar`

Reply:
> IMO this class doesn't need to exist, you can just select `.nx-btn` within `.nx-transfer-list__button-bar`

Sounds good! I removed it here: 8da7e29ba73263e6ce9f96e12d36ba9915305d0f